config/environments/development.rb

```diff
@@ -32,6 +32,7 @@
   config.action_mailer.raise_delivery_errors = false
 
   config.action_mailer.perform_caching = false
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
@Br3nda we could change the port to something like `Rails::Server.new.options[:Port]`sometimes Im running multiple instances or react on port 3000, this should get the current port.